### ISSUE TITLE
PAC com contrato não aparece.

### ISSUE THREAD

**Issue by r1tt3r** [pedro-teixeira/correios#245]
Realizei a atualização do modulo para versão 4.8.0, em serviços marquei:

SEDEX CONTRATO AGENCIA
PAC CONTRATO AGENCIA

Porem quando consulto o frete aprece apenas o SEDEX. Quando seleciono PAC SEM CONTRATO ele consegue calcular normalmente.

Alguém pode me confirmar se isso está relacionado ao contrato com os Correios ou algum outro erro ?

Obrigado.

**Comment by rafaelpatro** [comment]
Veja se a opção _Serviços por Produto_ está ativa. Se estiver ativa, configure os serviços em cada produto. Senão, desative.

**Comment by arnonfumagalli** [comment]
Quando o cliente tem contrato, o PAC não é listado para dentro do mesmo Estado de envio. Em clientes sem contrato isso não ocorre.

**Comment by r1tt3r** [comment]
@arnonfumagalli observei a mesma coisa, dexei o PAC sem contrato e Sedex Com Contrato, mas acaba que Sedex fica mais barato. Sabe se isso é um erro dos Correios ?

**Comment by arnonfumagalli** [comment]
Exato, mas não é erro. Os Correios alteraram isso recentemente, é uma nova regra nacional. Parece que como anteriormente pra dentro do estado o preço ficava muito próximo do Sedex, eles acabaram suspendendo o Pac.

**Comment by r1tt3r** [comment]
@arnonfumagalli Entendi ! O problema é a regra de Frete Gratis, se não existir PAC nas opções não vai ter frete grátis.

**Comment by arnonfumagalli** [comment]
Exatamente, hehe. Abri um outro tópico (#248) recentemente sobre isso. Infelizmente sem saída por enquanto. Nos nossos clientes optamos por remover o contrato enquanto isso não é resolvido. Assim funciona normalmente.
